py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -189,26 +191,97 @@ def native_group_norm(
 
     weight = get_trt_tensor(ctx, weight, f"{name}_weight")
     bias = get_trt_tensor(ctx, bias, f"{name}_bias")
```

Comment:
Per [the schema](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml#L2965), does this work for `weight==None` or `bias==None`?

Reply:
Nice catch. Added that case